stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/shared/ChatInfoSharedAttachmentsViewModel.kt

```diff
@@ -49,38 +51,33 @@ class ChatInfoSharedAttachmentsViewModel(
     }
 
     private suspend fun fetchAttachments() {
-        val currentState = state.value!!
-        val result = if (attachmentsType == AttachmentsType.FILES) {
-            channelClient.getFileAttachments(offset = currentState.results.size, limit = QUERY_LIMIT).await()
-        } else {
-            channelClient.getImageAttachments(offset = currentState.results.size, limit = QUERY_LIMIT).await()
-        }
-
+        val result = channelClient.getMessagesWithAttachments(messages.size, limit = QUERY_LIMIT, attachmentsType.requestTypeKey).await()
```

Comment:
It would be a good idea to name all parameters explicitly here, just like `limit`, so that we don't break this if we change the API we're calling into